.github/workflows/tests.yml

```diff
@@ -90,30 +90,63 @@ jobs:
         sudo microk8s.start
 
         # https://microk8s.io/docs/addons
-        sudo microk8s.enable dns
+        sudo microk8s.enable dns storage ingress
     - name: Wait for Kubernetes to be ready
       timeout-minutes: 15
       run: |
+        sudo microk8s.status
         while [[ ! $(sudo microk8s.kubectl cluster-info) ]] ; do
           echo "kubernetes not ready yet..." >&2
           sleep 1
         done
+        sudo microk8s.config > "${KUBECONFIG}"
+
     - name: Install CertManager v${{ matrix.certManager }}
       run: |
         # Try the recet way to install crd or fallback to the old one
         version='${{ matrix.certManager }}'
         shortVersion="${version%.*}"
-        sudo microk8s.kubectl apply -f "https://github.com/jetstack/cert-manager/releases/download/v${version}/cert-manager.crds.yaml" ||
-        sudo microk8s.kubectl apply -f "https://raw.githubusercontent.com/jetstack/cert-manager/release-${shortVersion}/deploy/manifests/00-crds.yaml"
+        sudo microk8s.kubectl apply -f "https://github.com/jetstack/cert-manager/releases/download/v${version}/cert-manager.yaml"
     - uses: actions/checkout@v2
     - name: go tests
       run: |
-        export KUBECONFIG="$(realpath "$(mktemp kubeconfig.XXXXX)")"
-        export USE_EXISTING_CLUSTER="true"
+        make go-test
 
-        sudo microk8s.config > "${KUBECONFIG}"
+    - name: install harbor-operator
+      run: |
+        set -ex
+        # make helm-install
+        sed -i '1inamespace: harbor-op' config/default/kustomization.yaml
```

Comment:
If we add the default namespace to Kustomization file, we may not need to do `sed` anymore, right?

Reply:
fix